GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -87,3 +97,8 @@ public final class StoryboardParser {
     }
   }
 }
+
+extension StoryboardParser.Segue: Equatable { }
+func ==(lhs: StoryboardParser.Segue, rhs: StoryboardParser.Segue) -> Bool {
+  return lhs.segueID == rhs.segueID
```

Comment:
Yep, good catch. There were no mentions of customClass in default template, so I've just ignored it which is not right. I'll address this.